lib/lsperfm/core/stats.rb

```diff
@@ -0,0 +1,64 @@
+# encoding: utf-8
+require "thread"
+
+Thread.abort_on_exception = true
+
+module LogStash::PerfM
```

Comment:
A lot of core methods mention "test", the gem description and summary has also "test", I'd call the top level module LogStash::PerformanceTest

But what was the M in PerfM for? just expand that and we move forward?